src/signals/incident-management/containers/IncidentDetail/components/StatusForm/StatusForm.tsx

```diff
@@ -144,9 +144,16 @@ const StatusForm: FunctionComponent<StatusFormProps> = ({
   const handleSubmit = useCallback(
     (event) => {
       event.preventDefault()
-
       const textValue = state.text.value || state.text.defaultValue
 
+      const doEmailTemplate =
```

Comment:
maybe change to hasSlugOverige

Reply:
Changed it to hasSlugsOverigAndStatusAfgehandeld